packages/crud/src/vaadin-crud.js

```diff
@@ -1264,7 +1265,9 @@ class Crud extends ControllerMixin(ElementMixin(ThemableMixin(PolymerElement)))
           this.items.push(item);
         }
       } else {
-        this.editedItem ||= {};
+        if (!this.editedItem) {
```

Comment:
I tried to replace `Object.assign` with setting an `editedItem` using spread, but this triggers an observer and causes a few tests to fail. So let's keep it as is and consider changing this logic later.